src/components/media-types/container.js

```diff
@@ -18,17 +18,43 @@ export const Container = ({ children = null, interactive }) => {
   })
 
   const toggleFullScreen = () => {
-    if (!document.fullscreenElement) {
-      document.documentElement.requestFullscreen()
+    // iPhone doesn't support Fullscreen API
+    // but we can toggle fullscreen class manually
+    const iPhone = /iPhone|iPod/.test(navigator.userAgent) && !window.MSStream
```

Comment:
maybe iPad.. ? :)

Reply:
ipad works actually, its just iphone that doesn't work with fullscreen API. tested here